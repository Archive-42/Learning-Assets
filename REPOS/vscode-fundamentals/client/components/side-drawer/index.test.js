// @ts-nocheck
import React from 'react';
import SideDrawer from './index';
import renderer from 'react-test-renderer';

test('SideDrawer renders as expected when drawer is showing', () => {

  const component = renderer.create(
    <SideDrawer drawerShowing={true}></SideDrawer>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('SideDrawer renders as expected when drawer is hiding', () => {

  const component = renderer.create(
    <SideDrawer drawerShowing={false}></SideDrawer>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
