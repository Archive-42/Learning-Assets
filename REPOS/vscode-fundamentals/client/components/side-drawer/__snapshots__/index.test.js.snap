// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideDrawer renders as expected when drawer is hiding 1`] = `
<div
  className="sidedrawer mui--no-user-select left"
/>
`;

exports[`SideDrawer renders as expected when drawer is showing 1`] = `
<div
  className="sidedrawer mui--no-user-select left active"
/>
`;
